CanonicalLtd/gypsy-danger
# Add apps

*TO QA THIS you'll need to reload the whole db*

This adds support for tracking hits of applications to the db. We add an `application_hits` db table with the charmid, charm name, owner, series, along with uuid/day. 

QA:

You should rebuild the python env so that you get the new dependency. 

    make clean-all
    make 
    python longrunning/longrunning.py initdb
    ....
    
    sqlite3 models.db
    .tables
    SELECT * FROM application_hits LIMIT 50;
